### PR TITLE
SC-6026 New Alert docs

### PR DESCRIPTION
**Pull request by mbonaci**

### CODE REVIEW

File: docs/alerts/index.md [hunk 1]
```diff
@@ -228,3 +285,107 @@ An email notification hook is created automatically during signup.  Additional n
...
+
+An email notification hook is created automatically during signup. Additional notification hooks can be created on the Notification Hooks view. A convenient way to navigate here is using the shortcut on top of the Alert Rules view.
+
+![image alt text](../images/alerts/image_10.png)
+
+It’s a shortcut for Integrations > Notification Hooks.
+
+![image alt text](../images/alerts/image_11.png)
+
+### Account-default hooks
```
Comment (adnanrahic) [inline docs/alerts/index.md]: This section has a dedicated file called `account-default-hooks.md` in the same directory as this `index.md`. This section should be moved there.

File: docs/alerts/index.md [hunk 2]
```diff
@@ -1,44 +1,101 @@
-title: Alerting on logs and metrics
```
Comment (adnanrahic) [inline docs/alerts/index.md]: Accidentally removed `title`?

**Comment by mbonaci** [comment]
@adnanrahic try when you can

**Comment by adnanrahic** [comment]
Okay. Did you try with `virtualenv`?

**Pull request merged**